src/pip/_internal/commands/__init__.py

```diff
@@ -3,57 +3,117 @@
 
+
+commands_dict = {
+    'check': (
+        'pip._internal.commands.check', 'CheckCommand',
+        'Verify installed packages have compatible dependencies.',
+    ),
+    'completion': (
+        'pip._internal.commands.completion', 'CompletionCommand',
+        'A helper command used for command completion.',
+    ),
+    'config': (
+        'pip._internal.commands.configuration', 'ConfigurationCommand',
+        'Manage local and global configuration.',
+    ),
+    'debug': (
+        'pip._internal.commands.debug', 'DebugCommand',
+        'Show information useful for debugging.',
+    ),
+    'download': (
+        'pip._internal.commands.download', 'DownloadCommand',
+        'Download packages.',
+    ),
+    'freeze': (
+        'pip._internal.commands.freeze', 'FreezeCommand',
+        'Output installed packages in requirements format.',
+    ),
+    'hash': (
+        'pip._internal.commands.hash', 'HashCommand',
+        'Compute hashes of package archives.',
+    ),
+    'help': (
+        'pip._internal.commands.help', 'HelpCommand',
+        'Show help for commands.',
+    ),
+    'list': (
+        'pip._internal.commands.list', 'ListCommand',
+        'List installed packages.',
+    ),
+    'install': (
+        'pip._internal.commands.install', 'InstallCommand',
+        'Install packages.',
+    ),
+    'search': (
+        'pip._internal.commands.search', 'SearchCommand',
+        'Search PyPI for packages.',
+    ),
+    'show': (
+        'pip._internal.commands.show', 'ShowCommand',
+        'Show information about installed packages.',
+    ),
+    'uninstall': (
+        'pip._internal.commands.uninstall', 'UninstallCommand',
+        'Uninstall packages.',
+    ),
+    'wheel': (
+        'pip._internal.commands.wheel', 'WheelCommand',
+        'Build wheels from your requirements.',
+    ),
+}  # type: Dict[str, Tuple[str, str, str]]
+
+
 commands_order = [
-    InstallCommand,
-    DownloadCommand,
-    UninstallCommand,
-    FreezeCommand,
-    ListCommand,
-    ShowCommand,
-    CheckCommand,
-    ConfigurationCommand,
-    SearchCommand,
-    WheelCommand,
-    HashCommand,
-    CompletionCommand,
-    DebugCommand,
-    HelpCommand,
-]  # type: List[Type[Command]]
-
-commands_dict = {c.name: c for c in commands_order}
+    'install',
+    'download',
+    'uninstall',
+    'freeze',
+    'list',
+    'show',
+    'check',
+    'config',
+    'search',
+    'wheel',
+    'hash',
+    'completion',
+    'debug',
+    'help',
+]  # type: List[str]
+
+
+def create_command(name, **kwargs):
+    # type: (str, **Any) -> Command
+    """
+    Create an instance of the Command class with the given name.
+    """
+    module_path, class_name, summary = commands_dict[name]
+    module = importlib.import_module(module_path)
+    command_class = getattr(module, class_name)
+    command = command_class(name=name, summary=summary, **kwargs)
+
+    return command
 
 
 def get_summaries(ordered=True):
```

Comment:
It looks like `get_summaries(ordered=False)` is never called so we could drop the arg...